javascripts/components/cardButtons.js

```diff
@@ -0,0 +1,133 @@
+
+import { singleDino, deadDino } from "./addDinos.js";
+import{ adventureRandomizer } from './adventuresList.js' 
+import {kennel, hospital, graveyard} from './../../assets/data/dinoData.js'
+
+import { infoModal } from './infoModal.js'
+
+const feedButton = (array, index, item) => {
+  $(`#feed-${item.id}`).on("click", () => {
+    if (item.health < 100) {
+      item.health += 10;
+      if (item.health > 100) {
+        item.health = 100;
+      }
+      $(`#progressBar-${item.id}`).html(``);
+      $(`#progressBar-${item.id}`).html(
+        ` <div class="progress-bar progress-bar-striped ${
+          item.health > 50 ? "bg-success" : "bg-warning"
+        } progress-bar-animated" style="width: ${
+          item.health
+        }%" id="progress-${index}" role="progressbar" aria-valuenow="0" aria-valuemin="0" aria-valuemax="100"><p class="health-status">${
+          item.health
+        }%</p></div>`
+      );
+      
+      sorter(array, item, index);
```

Comment:
smart! I am wondering though, if you are importing the arrays, then why you need to pass them through the button functions rather than evaluating the imported arrays here? Maybe see if you can pass the array name as a string instead of the entire data element?